src/main/java/org/opensearch/neuralsearch/sparse/common/SparseConstants.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.neuralsearch.sparse.common;
+
+/**
+ * Constants used throughout the sparse vector search implementation.
+ * Contains field names, configuration parameters, and default values for SEISMIC algorithm.
+ */
+public final class SparseConstants {
+    public static final String NAME_FIELD = "name";
+    public static final String PARAMETERS_FIELD = "parameters";
+    public static final String N_POSTINGS_FIELD = "n_postings";
+    public static final String SUMMARY_PRUNE_RATIO_FIELD = "summary_prune_ratio";
+    public static final String SEISMIC = "seismic";
+    public static final String CLUSTER_RATIO_FIELD = "cluster_ratio";
+    public static final String APPROXIMATE_THRESHOLD_FIELD = "approximate_threshold";
+    public static final String INDEX_THREAD_QTY = "indexThreadQty";
+
+    /**
+     * SEISMIC algorithm configuration constants and default values.
+     */
+    public static final class Seismic {
+        public static final int DEFAULT_N_POSTINGS = -1;
+        public static final float DEFAULT_SUMMARY_PRUNE_RATIO = 0.4f;
+        public static final float DEFAULT_CLUSTER_RATIO = 0.1f;
+        public static final int DEFAULT_APPROXIMATE_THRESHOLD = 1000000;
```

Comment:
Should this be 100000?

Reply:
It's 1000k by default, not 100k.

Reply:
it's better to track it somewhere else like adding a task in a project